Jenkinsfile

```diff
@@ -278,33 +278,41 @@ pipeline {
                 DOCKER_GCR = credentials("zeebe-gcr-serviceaccount-json")
                 ZEEBE_AUTHORIZATION_SERVER_URL = 'https://login.cloud.ultrawombat.com/oauth/token'
                 ZEEBE_CLIENT_ID = 'W5a4JUc3I1NIetNnodo3YTvdsRIFb12w'
-                QA_RUN_VARIABLES = "{\"zeebeImage\": \"${env.IMAGE}:${env.TAG}\", \"generationTemplate\": \"${params.GENERATION_TEMPLATE}\", \"channel\": \"Internal Dev\", " +
-                    "\"branch\": \"${env.BRANCH_NAME}\", \"build\": \"${currentBuild.absoluteUrl}\"}"
+                QA_RUN_VARIABLES = "{\"zeebeImage\": \"${env.IMAGE}:${env.TAG}\", \"generationTemplate\": \"${params.GENERATION_TEMPLATE}\", " +
+                                    "\"channel\": \"Internal Dev\", \"branch\": \"${env.BRANCH_NAME}\", \"build\": \"${currentBuild.absoluteUrl}\", " +
+                                    "\"businessKey\": \"${currentBuild.absoluteUrl}\", \"processId\": \"qa-protocol\"}"
+                BUSINESS_KEY = "${currentBuild.absoluteUrl}"
             }
 
             steps {
-                timeout(time: shortTimeoutMinutes, unit: 'MINUTES') {
```

Comment:
Is dropping the time out on purpose?

Reply:
Yes, it is intended. Before this was the timeout for starting the QA run. And then there was a second step to wait for user input, which had no timeout. Now the waiting for the result is part of this process. Currently, the QA run has to wait until all analyses have been completed, so it can take an arbitrary number of time.
Later this week we hopefully have an implementation that sends the result to Jenkins as soon as possible, without waiting for the analysis to be completed. At this point we could add a generous timeout again here.